src/Chapter2.hs

```diff
@@ -621,7 +634,10 @@ Write a function that takes elements of a list only on even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: [a] -> [a]
+takeEven [] = []
+takeEven (x1:_:xs) = x1 : takeEven xs
+takeEven (x1:xs) = x1 : takeEven xs
```

Comment:
Here actually only one pattern left: singleton list. So you can write it explicitly to be sure that you've covered all cases :slightly_smiling_face: 
```suggestion
takeEven [x] = [x]
```